assets/js/modules/analytics-4/components/dashboard/ActivationBanner/SetupBanner.js

```diff
@@ -137,47 +164,56 @@ export default function SetupBanner( { onCTAClick } ) {
 				) }
 			</div>
 		);
-	} else if ( existingTag ) {
-		title = __(
-			'No existing Google Analytics 4 property found, Site Kit will help you create a new one and insert it on your site',
-			'google-site-kit'
-		);
-		ctaLabel = __( 'Create property', 'google-site-kit' );
-		footer = sprintf(
-			/* translators: %s: The existing tag ID. */
-			__(
-				'A GA4 tag %s is found on this site but this property is not associated with your Google Analytics account. You can always add/edit this in the Site Kit Settings.',
-				'google-site-kit'
-			),
-			existingTag
-		);
 	} else {
-		title = __(
-			'No existing Google Analytics 4 property found, Site Kit will help you create a new one and insert it on your site',
-			'google-site-kit'
-		);
-		ctaLabel = __( 'Create property', 'google-site-kit' );
-		footer = __(
-			'You can always add/edit this in the Site Kit Settings.',
-			'google-site-kit'
-		);
+		selectProperty( PROPERTY_CREATE );
+
+		if ( existingTag ) {
+			title = __(
+				'No existing Google Analytics 4 property found, Site Kit will help you create a new one and insert it on your site',
+				'google-site-kit'
+			);
+			ctaLabel = __( 'Create property', 'google-site-kit' );
+			footer = sprintf(
+				/* translators: %s: The existing tag ID. */
+				__(
+					'A GA4 tag %s is found on this site but this property is not associated with your Google Analytics account. You can always add/edit this in the Site Kit Settings.',
+					'google-site-kit'
+				),
+				existingTag
+			);
+		} else {
+			title = __(
+				'No existing Google Analytics 4 property found, Site Kit will help you create a new one and insert it on your site',
+				'google-site-kit'
+			);
+			ctaLabel = __( 'Create property', 'google-site-kit' );
+			footer = __(
+				'You can always add/edit this in the Site Kit Settings.',
+				'google-site-kit'
+			);
+		}
 	}
 
 	return (
 		<BannerNotification
 			id="ga4-activation-banner"
 			className="googlesitekit-ga4-setup-banner"
 			title={ title }
-			ctaLabel={ ctaLabel }
-			ctaLink={ onCTAClick ? '#' : null }
-			onCTAClick={ onCTAClick }
+			ctaComponent={
+				! ga4MeasurementID && (
```

Comment:
As discussed, I am not so sure this condition is required, it seems we should always be rendering the CTA here...

Reply:
I initially thought we might need a different functionality for the `Connect` action instead of `submitChanges`. I've removed this, thank you!